cypress/e2e/ui/Settings/Application-Settings/tenant.cy.js

```diff
@@ -0,0 +1,634 @@
+        // Open the tenant edit form
+        cy.toolbar(configToolbarButton, editTenantConfigOption);
+      });
+
+      it('Validate Edit tenant form elements', () => {
+        validateFormElements();
+      });
+
+      it('Validate Reset & Cancel buttons on Edit tenant form', () => {
+        // Update name & description fields
+        updateNameAndDescription(editedTenantNameValue, editedDescriptionValue);
+        // Reset the form
+        resetForm('button');
+        // Confirm name and description has old values
+        verifyNameAndDescriptionValues(
+          initialParentTenantName,
+          initialParentTenantDescription
+        );
+        // Cancel the edit form
+        cancelForm();
+      });
+
+      it('Validate Edit function', () => {
+        // Edit name & description
+        updateNameAndDescription(editedTenantNameValue, editedDescriptionValue);
+        // Save the form
+        saveForm();
+      });
+
+      afterEach(() => {
+        confirmUiNavigation(() => resetParentTenantForm());
+      });
+    });
+
+    describe('Validate Add Project to parent tenant', () => {
+      it('Validate Add Project function', () => {
+        // Add Project to the parent tenant
+        addProjectToTenant();
+      });
+
+      after(() => {
+        confirmUiNavigation(() => deleteAccordionItem(projectNameValue));
+      });
+    });
+
+    describe('Validate Manage Quotas in parent tenant', () => {
+      beforeEach(() => {
+        // Open the Manage Quotas form
+        cy.toolbar(configToolbarButton, manageQuotasConfigOption);
+      });
+
+      it('Validate Reset & Cancel buttons in Manage Quotas form', () => {
+        // Check if Reset button is disabled initially
+        cy.getFormFooterButtonByType(resetButton, 'reset').should(
+          'be.disabled'
+        );
+        // Editing the quota table
+        editQuotasTable(allocatedStorageQuota);
+        // Reset the form
+        resetForm();
+
+        // Cancel the Manage Quotas form
+        cancelForm();
+      });
+
+      it('Validate Manage Quotas function', () => {
+        // Enabling the quota: "Allocated Storage in GB" with value 10
+        editQuotasTable(allocatedStorageQuota, '10');
+        // Saving the form
+        saveForm();
+      });
+
+      after(() => {
+        // Resetting the quotas back to all disabled state
+        confirmUiNavigation(() => rollbackQuotas());
+      });
+    });
+  });
+
+  describe('Validate Child Tenant operations: Add, Edit, Add Project, Manage Quotas', () => {
+    describe('Validate Add child tenant function', () => {
+      beforeEach(() => {
+        // Open the Add child tenant form
+        cy.toolbar(configToolbarButton, addChildTenantConfigOption);
+      });
+
+      it('Validate Add child tenant form elements', () => {
+        validateFormElements(false);
+      });
+
+      it('Validate cancel button on Add child tenant form', () => {
+        // Cancel the add operation
+        cancelForm();
```

Comment:
we call cancelForm(); in several other places... I'm not sure it's worth testing on it's own here.  Maybe it can be tested in the test above: `validateFormElements`.  In other words, when validating the add child tenant form elements, I'm thinking the cancel button is part of a way to test the form.